content/en/docs/ambient/usage/add-workloads/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Add workloads to the mesh
+description: Understand how to add workloads to an ambient mesh.
+weight: 1
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+In most cases, a cluster administrator will deploy the Istio mesh infrastructure. Once Istio is successfully deployed in ambient mode, it will be transparently available to applications deployed by all users in namespaces that have been configured to use it.
```

Comment:
```suggestion
In most cases, a cluster administrator will deploy the Istio mesh infrastructure. Once Istio is successfully deployed with ambient mode enabled, the ambient {{< gloss >}}data plane{{< /gloss >}} will be transparently available to applications and namespaces in the cluster that have been configured to use it.
```

Reply:
I am mostly pushing the `data plane` verbiage because it makes statements like this clearer IMO - you install ambient _with support for_ the ambient data plane mode, and then you _may choose_ to use the ambient data plane mode, the sidecar data plane mode, or both, as a runtime concern.

"ambient mode" is a bit too fuzzy, we should probably be clear with each usage whether we are talking about "the install flag" or "the runtime mode" since they aren't the same thing.

(that being said, I am fine with eliding the difference where it makes grammatical sense)

Reply:
Good centi-thread on Slack: to summarise, I'm going to try a pass with

* "support for ambient mode" instead of "ambient mode enabled" (with a big change throughout docs/blogs) to be clearer
* "ambient data plane mode" on first use, or where required for disambiguation, "ambient mode" thereafter